test/unit-test/main.tf

```diff
@@ -58,6 +58,45 @@ module "s3_with_notification" {
 
 }
 
+#trivy:ignore:AVD-AWS-0086
+#trivy:ignore:AVD-AWS-0087
+#trivy:ignore:AVD-AWS-0088
+#trivy:ignore:AVD-AWS-0090
+#trivy:ignore:AVD-AWS-0091
+#trivy:ignore:AVD-AWS-0093
+#trivy:ignore:AVD-AWS-0094
+#trivy:ignore:AVD-AWS-0132
+resource "aws_s3_bucket" "non-modulised-bucket" {
+  #checkov:skip=CKV2_AWS_6: "Ensure that S3 bucket has a Public Access block - This is not needed in our tests"
+  #checkov:skip=CKV_AWS_18: "Ensure the S3 bucket has access logging enabled - This is not needed in our tests"
+  #checkov:skip=CKV_AWS_21: "Ensure all data stored in the S3 bucket have versioning enabled - This is not needed in our tests"
+  #checkov:skip=CKV2_AWS_61: "Ensure that an S3 bucket has a lifecycle configuration - This is not needed in our tests"
+  #checkov:skip=CKV2_AWS_62: "Ensure S3 buckets should have event notifications enabled - This is not needed in our tests"
+  #checkov:skip=CKV_AWS_144: "Ensure that S3 bucket has cross-region replication enabled - This is not needed in our tests"
+  #checkov:skip=CKV_AWS_145: "Ensure that S3 buckets are encrypted with KMS by default - This is not needed in our tests"
+  bucket = "log-test-bucket-051683332738327"
```

Comment:
`bucket_prefix = "log-test-bucket"` would keep resources here unique which might be better for unit tests?

Reply:
i tried this originally, but the problem is (i forgot it was the original problem i was tackling) if i try and call the bucket name via code in the same terraform plan: it fails (as there's a conditional on something that hasnt been generated yet), so this was the only way i could get it to work 😓

Reply:
OK! Makes total sense - thanks for the explanation :)